public/js/actions/user.js

```diff
@@ -127,3 +127,25 @@ export function userSignOut(jquery=$) {
     });
   };
 }
+
+
+export function getBraintreeToken() {
+  console.log('Requesting braintree token');
+  return dispatch => {
+    $.ajax({
+      method: 'post',
+      url: '/api/braintree/token/generate/',
+      context: this,
+    }).then(data => {
+      dispatch({
+        type: actionTypes.RX_BRAINTREE_TOKEN,
```

Comment:
I thought more about this and since it's the *result* of the action it doesn't make perfect sense to follow the action name in the same verb tense. That made me realize that `RX` stands for receive :) However, I think you could use `GOT_BRAINTREE_TOKEN` and get the same effect with less cryptic abbreviation.